buildkite/aggregate_incompatible_flags_test_result.py

```diff
@@ -196,14 +196,34 @@ def get_html_link_text(content, link):
 
 
 def print_flags_ready_to_flip(failed_jobs_per_flag, details_per_flag):
-    info_text = ["#### The following flags didn't break any passing jobs"]
+    info_text1 = ["#### The following flags didn't break any passing projects"]
     for flag in sorted(list(details_per_flag.keys())):
         if flag not in failed_jobs_per_flag:
             github_url = details_per_flag[flag].issue_url
-            info_text.append(f"* **{flag}** " + get_html_link_text(":github:", github_url))
-    if len(info_text) == 1:
-        return
-    print_info("flags_ready_to_flip", "success", info_text)
+            info_text1.append(f"* **{flag}** " + get_html_link_text(":github:", github_url))
```

Comment:
Here and in line 221: save the link text in a var and use a single format string instead of string concatenation.

Reply:
Thanks, done.